docs/modules/ROOT/pages/how-to/monitor-application.adoc

```diff
@@ -0,0 +1,232 @@
+<2> A selector that points to the service of the application
++
+TIP: Consult the https://docs.openshift.com/container-platform/4.11/rest_api/monitoring_apis/servicemonitor-monitoring-coreos-com-v1.html[ServiceMonitor API] for more configuration options.
+
+. View the collected metrics
++
+Navigate to the OpenShift Web Console https://console.CLUSTER.cloud/dev-monitoring/ns/<APP_NAMESPACE>/metrics.
++
+Select __Custom Query__ and enter `rate(http_requests_total[1m])`.
+You should see a graph showing the request rate to the example application.
++
+image::monitoring/custom-metrics-graph.png[Collected metrics on https://console.<CLUSTER>.cloud/dev-monitoring/ns/<APP_NAMESPACE>/metrics]
+
+== Create an Alert
+
+. Create the alert rule
++
+[source,shell]
+----
+kubectl -n "${APP_NAMESPACE}" apply -f - <<YAML
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: prometheus-example-alert
+spec:
+  groups:
+  - name: prometheus-example-alert
+    rules:
+    - alert: TooManyrequests
+      expr: rate(http_requests_total[1m]) > 1 <1>
+      labels:
+        severity: info
+      annotations:
+        summary: "There are a lot of requests to the application"
+YAML
+----
+<1> The alert rule will fire if the application receives more than 60 requests in one minute.
++
+TIP: Look at the https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/[upstream documentation] on writing your own alerting rules.
+
+. Verify that the alert was picked up by Prometheus
++
+Navigate to the OpenShift Web Console https://console.<CLUSTER>.cloud/dev-monitoring/ns/<APP_NAMESPACE>/alerts.
++
+image::monitoring/custom-metrics-alert-list.png[List of alerts https://console.<CLUSTER>.cloud/dev-monitoring/ns/<APP_NAMESPACE>/alerts]
++
+The alert should be displayed in the list of alerts.
++
+[TIP]
+====
+It may take a few minutes for the alert to show up.
+If it doesn't show up, that most likely means that the `PrometheusRule` CRD isn't well-formed and might have failed to apply, or that the PromQL expression isn't valid.
+
+Check if the `PrometheusRule` CRD was created successfully.
+
+[source,shell]
+----
+kubectl -n "${APP_NAMESPACE}" get prometheusrules
+----
+
+If this seems correct, check that the PromQL expression is valid by entering it in the UI.
+====
+
+. Check that the alert will fire by generating requests
++
+[source,shell]
+----
+for i in {1..70}
+do
+  curl https://$APP_NAMESPACE.apps.$ZONE.appuio.cloud
+done
+----
+Navigate to the OpenShift Web Console https://console.<CLUSTER>.cloud/dev-monitoring/ns/<APP_NAMESPACE>/alerts.
++
+The alert should now be firing.
++
+image::monitoring/custom-metrics-alert-fire.png[Firing alert https://console.<CLUSTER>.cloud/dev-monitoring/ns/<APP_NAMESPACE>/alerts]
++
+TIP: You can send the alert to a receiver such as email, PagerDuty, or Opsgenie.
+//TODO Fix link
+To do so follow the xref:how-to/alert-routing.adoc[Alert Routing How-to].
```

Comment:
Please make sure this reference is correct before merging. (This How-To was added before adding the Alert Routing How-To)